### PR TITLE
Groupings next

### PR DESCRIPTION
**Pull request by stdavis**
My Friday

### CODE REVIEW

File: src/reportParts/HazardUnit.js [hunk 1]
```diff
@@ -44,12 +44,16 @@ export default props => {
 
   return (
     <div className="unit">
-      <span ref={legend} className="unit__legend"></span>
```
Comment (steveoh) [inline src/reportParts/HazardUnit.js]: 😂

File: src/config.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+    otherDataTable: `${baseUrl}/Report_Tables_View/FeatureServer/7`,
+    reportTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/8`
   },
   queries: [
-    // ['Utah_Geologic_Hazards/FeatureServer/0', '???'], // flood canyon
+    ['Utah_Geologic_Hazards/FeatureServer/0', 'FCH'],
     ['Utah_Geologic_Hazards/FeatureServer/1', 'FLH'],
     ['Utah_Geologic_Hazards/FeatureServer/2', 'SGS'],
     ['Utah_Geologic_Hazards/FeatureServer/3', 'LSS'],
     // ['Utah_Geologic_Hazards/FeatureServer/4', '???'], // landslides
```
Comment (steveoh) [inline src/config.js]: https://github.com/agrc/ugs-hazards/issues/2#issuecomment-542264075

```suggestion
    ['Utah_Geologic_Hazards/FeatureServer/4', 'LSF'], // landslides
```

File: src/reportParts/SummaryPage.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  let table1Heading = 'Table 1';
+  let table2Heading = 'Table 2';
+  Object.keys(props).forEach(key => {
+    if (key.match(table1Heading)) {
+      table1Heading = props[key];
+    } else if (key.match(table2Heading)) {
+      table2Heading = props[key];
+    }
+  });
```
Comment (steveoh) [inline src/reportParts/SummaryPage.js]: this seems a bit strange. Is this code below the same and a little bit more straight forward?

```js
if (table1Heading in props) {
  table1Heading = props[table1Heading];
}

if (table2Heading in props) {
  table2Heading = props[table2Heading];
}
```

Comment (stdavis) [reply src/reportParts/SummaryPage.js]: Yes, that's much better. I forget that you can check for object keys that way. I'll update...

**Pull request merged**